app/templates/components/public/session-list.hbs

```diff
@@ -0,0 +1,8 @@
+{{#each-in sessions as |id trackSession|}}
```

Comment:
Crap. Did not notice that groupBy in the filtered output. We should remove that. No point grouping by track when we are filtering by track.